Makefile

```
@@ -0,0 +1,12 @@
+.PHONY: docs-main
+docs-main:
+	helm-docs -t README-main.md.gotmpl -g .
+
+.PHONY: docs-sub
+docs-sub:
+	helm-docs -t README-sub.md.gotmpl -g charts/agent,charts/server
+
+.PHONY: docs
+docs: docs-sub docs-main
```

I would for now stick to using a single template for all charts and `README-main.md.gotmpl` for the main / parent chart. We could test skipping the parent chart with the first command, but for now it should also work by simply overwriting it again by the second command. (I also tried moving all charts into a sub-directory with no success #43)

```suggestion
.PHONY: docs
docs:
	helm-docs # updates all charts including subcharts
	helm-docs -t README-main.md.gotmpl -g .
```